Find a functionally equivalent code:android.graphics.Paint.clearShadowLayer:COMMENT
Method Modifier: public      
Comment:/**
 * Clear the shadow layer.
 */

@@ -1,4 +1,3 @@
 {
-    hasShadow = false;
-    nSetShadowLayer(0, 0, 0, 0);
+    setShadowLayer(0, 0, 0, 0);
 }

